falcon/app.py

```diff
@@ -238,32 +240,6 @@ def __init__(self, media_type=DEFAULT_MEDIA_TYPE,
         self.add_error_handler(falcon.HTTPError, self._http_error_handler)
         self.add_error_handler(falcon.HTTPStatus, self._http_status_handler)
 
-    def add_middleware(self, middleware):
```

Comment:
I reordered a few methods to make them easier to find when going back and forth between `falcon.App` and `falcon.asgi.App`.